src/js/game_logic.js

```diff
@@ -1,3 +1,5 @@
+const PLAY_TIME_SECONDS = 5
```

Comment:
Any particular reason for all-caps snake_case? Could we rewrite it to fit the current coding style?

Reply:
I've always name constant that I can actively change to alter the app's behavior to ALL_CAPS.
In this case, it's the game play time.
However, I can totally update it to match everything else.